lisp/mastodon-media.el

```diff
@@ -42,23 +42,24 @@
   "Takes a URL and return an image."
   (let ((buffer (url-retrieve-synchronously url)))
     (unwind-protect
-	 (let ((data (with-current-buffer buffer
-		       (goto-char (point-min))
-		       (search-forward "\n\n")
-		       (buffer-substring (point) (point-max)))))
-	   (insert "\n")
-	   (insert-image (create-image data nil t)))
+        (let ((data (with-current-buffer buffer
+                      (goto-char (point-min))
+                      (search-forward "\n\n")
+                      (buffer-substring (point) (point-max)))))
+          (insert "\n")
+          (insert-image (create-image data nil t)))
       (kill-buffer buffer))))
 
-(defun mastodon-media--select-media-line (start)
-  "Returns the list of line coordinates of a line that
+(defun mastodon-media--select-next-media-line ()
+  "Find coordinates of a line that contains `Media_Links::'
 
-contains `Media_Links::'."
-  (when start (goto-char (point-min)))
-  (search-forward-regexp "Media_Link::" nil nil nil)
-  (let ((start (progn (move-beginning-of-line '()) (point)))
-	(end (progn (move-end-of-line '()) (point))))
-    (list start end)))
+Returns the cons of (`start' . `end') points of that line or nil no
+more media links were found."
+  (let ((found? (search-forward-regexp "Media_Link::" nil t)))
```

Comment:
Change `found?` to `foundp` to match convention.

https://github.com/bbatsov/emacs-lisp-style-guide#naming